packages/core/src/Stores/ui-store.js

```diff
@@ -668,8 +668,10 @@ export default class UIStore extends BaseStore {
     }
 
     @action.bound
-    toggleWelcomeModal(is_visible = !this.is_welcome_modal_visible) {
+    toggleWelcomeModal(is_visible = !this.is_welcome_modal_visible, should_persist) {
```

Comment:
default `should_persist` to false and make this an object instead of sending `true, false`?